poko-compiler-plugin/src/main/kotlin/dev/drewhamilton/poko/ir/hashCodeGeneration.kt

```diff
@@ -144,6 +146,14 @@ private fun IrBlockBodyBuilder.getHashCodeOf(
     if (property.type.isInt()) {
         return value
     }
+    if (property.type.isUInt()) {
+        val uintToInt = referenceClass(StandardNames.FqNames.uInt)!!
+            .functions
+            .first { it.owner.name.asString() == "toInt" }
```

Comment:
```suggestion
            .single { it.owner.name.asString() == "toInt" }
```
I'd rather enforce the expectation that there's only one `UInt.toInt` than trust the hunch they'll never add an overload.